### DISCUSSION TITLE
Dark mode

### DISCUSSION THREAD

**Discussion by colivinggit** [Xammie/mailbook#74] [Ideas]
Any chance to show how dark mode will look like?

**Comment by Xammie** [comment]
Are you suggesting a dark/light mode toggle? How do you currently support light and dark mode in your mail templates? If you could give some examples that would be great.

I'm not really sure how that would technically work in Mailbook though because I can't just change `@media (prefers-color-scheme: dark)` to be true or false. What do you think?

**Comment by colivinggit** [comment]
I'm not sure. I get people telling me that the emails look bad in dark mode. 
I don't use emails in dark mode mysefl.

**Reply by Xammie** [reply]
Maybe they are referring to the automatic dark-mode that some email clients have?
- https://stackoverflow.com/a/76382430/5228838
- https://onesignal.com/blog/how-to-optimize-emails-for-dark-mode/

I suppose the full invert could be recreated in mailbook by using a CSS filter when you want to display dark mode. Gonna think about if this would be a useful feature.
